packages/local-notifications/index.android.ts

```diff
@@ -206,45 +213,45 @@ export class LocalNotificationsImpl extends LocalNotificationsCommon implements
 
 			const context = Utils.android.getApplicationContext();
 			const resources = context.getResources();
-			const scheduledIds: Array<number> = [];
+			const scheduledIds: number[] = [];
 
 			// TODO: All these changes in the options (other than setting the ID) should rather be done in Java so that
 			// the persisted options are exactly like the original ones.
-
-			for (let n in scheduleOptions) {
-				const triggers: Array<Record<string, any>> = [];
-				const options = LocalNotificationsImpl.merge(scheduleOptions[n], LocalNotificationsImpl.defaults);
-				const [interval, ticks] = !!options.interval && options.interval.constructor === Object ? (Object.entries(options.interval || {}).shift() as [ScheduleInterval, number]) || [] : ([options.interval] as [ScheduleInterval]);
-
-				LocalNotificationsImpl.ensureID(options);
-
-				options.atTime = options.at ? options.at.getTime() : -1;
-				if (interval) {
-					options.atTime = Date.now() + options.repeatInterval;
+			for (const s of scheduleOptions) {
+				const entry = LocalNotificationsImpl.createScheduleEntry(s);
+				// Some properties should not be inherited by options
+				const nativeOptions: ScheduleNativeOptions = { ...entry, color: undefined, notificationLed: undefined };
```

Comment:
Not at all. The `nativeOptions` object is meant for being sent to the native side as a json string.
In line 223, we mark color and notificationLed as undefined to:
- Prevent them from being read from native lib in case they're not set below (because they're set below)
- Correct type warnings